### PR TITLE
Add description for face-detection-retail-0005

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
@sovrasov it seems you also need to add new file into index.md
```
15:42:29  + /home/jenkins/workspace/omz/precommit/documentation_ubuntu16/jenkins/omz/scripts/documentation_check.py src/intel_models
15:42:29  face-detection-retail-0005 not in index.md file
```

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
@sovrasov could you please check other models descriptions to see what is the correct link to benchmark data? I think benchmark data not available at the moment, so other models have 'Performance' section just empty.
```
19:03:04  FILE: face-detection-retail-0005/description/face-detection-retail-0005.md
19:03:31  [✖] https://software.intel.com/en-us/openvino-toolkit/benchmarks
19:03:31  
19:03:31  ERROR: dead links found!
```

File: intel_models/face-detection-retail-0005/description/face-detection-retail-0005.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`],
+   where:
+
+    - `image_id` - ID of the image in the batch
+    - `label` - predicted class ID
+    - `conf` - confidence for the predicted class
+    - (`x_min`, `y_min`) - coordinates of the top left bounding box corner
+    - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner.
+
+## Legal Information
```
Comment (IRDonch) [inline intel_models/face-detection-retail-0005/description/face-detection-retail-0005.md]: Where is the legal information?

**Comment by IRDonch** [comment]
Jenkins: ok to test

File: intel_models/face-detection-retail-0005/description/face-detection-retail-0005.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+![](./face-detection-retail-0001.png)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP ([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/)) | 84.52%                  |
+| GFlops                                                        | 0.982                   |
+| MParams                                                       | 1.021                   |
+| Source framework                                              | Pytorch*                |
```
Comment (IRDonch) [inline intel_models/face-detection-retail-0005/description/face-detection-retail-0005.md]: I believe the correct spelling is PyTorch.

**Pull request merged**